### PR TITLE
Renew the readme

### PR DESCRIPTION
**Pull request by beam2d**
I rewrote the readme for v2. It is simpler than the current one and easier to find the appropriate place to get the information that users want (which is the most important thing that readme should provide).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,120 +4,66 @@
...
-Anaconda distribution includes this package.
-If you are using another Python distribution, use either of the following commands to install libhdf5 depending on your Linux environment:
+## Contribution
 
-```
-apt-get install libhdf5-dev
-yum install hdf5-devel
-```
+Any contributions to Chainer are welcome!
+If you want to file an issue or send a pull request, [follow the contribution guide](https://docs.chainer.org/contribution.html).
```
Comment (mitmul) [inline README.md]: "please follow" would be more polite?

File: README.md [hunk 2]
```diff
@@ -4,120 +4,66 @@
...
 
-## Installation
+[Website](http://chainer.org/)
+| [Docs](http://docs.chainer.org/en/stable/)
+| [Install Guide](http://docs.chainer.org/en/stable/install.html)
+| [Tutorial](http://docs.chainer.org/en/stable/tutorial/)
+| Examples ([Official](https://github.com/pfnet/chainer/blob/master/examples), [External](https://github.com/pfnet/chainer/wiki/External-examples))
+| Forum ([en](https://groups.google.com/forum/#!forum/chainer), [ja](https://groups.google.com/forum/#!forum/chainer-jp))
+| Slack ([en](https://bit.ly/chainer-slack), [ja](https://bit.ly/chainer-jp-slack))
+| Twitter ([en](https://twitter.com/ChainerOfficial), [ja](https://twitter.com/ChainerJP))
```
Comment (mitmul) [inline README.md]: How about making these links bold style? It could be hard for some people to notice.

**Comment by beam2d** [comment]
Fixed following your suggestions, and merged the latest v2.

**Comment by mitmul** [comment]
LGTM

**Pull request merged**